src/main/java/hudson/plugins/sshslaves/SSHConnector.java

```diff
@@ -67,242 +66,187 @@
     /**
      * Field port
      */
-    public final int port;
+    public int port;
 
     /**
      * The id of the credentials to use.
      */
     private String credentialsId;
 
-    /**
-     * Transient stash of the credentials to use, mostly just for providing floating user object.
-     */
-    private transient StandardUsernameCredentials credentials;
-
-    /**
-     * Field username
-     *
-     * @deprecated
-     */
-    @Deprecated
-    public transient String username;
-
-    /**
-     * Field password
-     *
-     * @deprecated
-     */
-    @Deprecated
-    public transient Secret password;
-
-    /**
-     * File path of the private key.
-     *
-     * @deprecated
-     */
-    @Deprecated
-    public transient String privatekey;
-
     /**
      * Field jvmOptions.
      */
-    public final String jvmOptions;
+    private String jvmOptions;
 
     /**
      * Field javaPath.
      */
-    public final String javaPath;
-
-
-    /**
-     * Field jdk
-     */
-    public final JDKInstaller jdkInstaller;
+    private String javaPath;
 
     /**
      * Field prefixStartSlaveCmd.
      */
-    public final String prefixStartSlaveCmd;
+    private String prefixStartSlaveCmd;
 
     /**
      * Field suffixStartSlaveCmd.
      */
-    public final String suffixStartSlaveCmd;
+    private String suffixStartSlaveCmd;
 
     /**
      *  Field launchTimeoutSeconds.
      */
-    public final Integer launchTimeoutSeconds;
+    private Integer launchTimeoutSeconds;
 
     /**
      *  Field maxNumRetries.
      */
-    public final Integer maxNumRetries;
+    private Integer maxNumRetries;
 
     /**
      *  Field retryWaitTime.
      */
-    public final Integer retryWaitTime;
+    private Integer retryWaitTime;
     
-    private final SshHostKeyVerificationStrategy sshHostKeyVerificationStrategy;
-
-    public StandardUsernameCredentials getCredentials() {
```

Comment:
Not deprecated method was removed

Reply:
the constructors that use it was deprecated for months